jmix-ui/ui/src/main/java/io/jmix/ui/UiProperties.java

```diff
@@ -135,6 +135,11 @@ public class UiProperties {
      */
     int mainTabCaptionLength;
 
+    /**
+     * Reinitialize session after login to protect from Session Fixation attacks.
```

Comment:
"Whether to reinitialize a session..." - like for other boolean attributes. Also, why Session Fixation start with capital letters?

Reply:
Got inspired from CUBA implementation, will fix that